docs/api/v3.rst

```diff
@@ -103,7 +103,11 @@ Projects list
             "count": 25,
             "next": "/api/v3/projects/?limit=10&offset=10",
             "previous": null,
-            "results": ["PROJECT"]
+            "results": [{
+                "id": 12345,
+                "name": "Pip",
+                ...
+            }]
```

Comment:
Is this more clear in the end? I'm not sure that mentioning just 2 fields will make people realize that it's a project (with all its fields) in all the endpoints. What are we using in the rest of the docs? We should be consistent through the whole document.

Also, maybe we want to be more explicit here saying something like "Project object" or "List of Project objects" maybe with a link to the Project Details section if possible.